lib/matplotlib/path.py

```diff
@@ -338,7 +339,7 @@ def make_compound_path(cls, *args):
         for path in args:
             if path.codes is None:
                 codes[i] = cls.MOVETO
-                codes[i + 1:i + len(path.vertices)] = cls.LINETO
+                codes[i + 1:i + len(codes)] = cls.LINETO
```

Comment:
Is that right? `len(codes)` is the entirety of `codes`, so you might as well leave out the upper limit.

Reply:
you're right, actually, this was the result of a bad automatic rebase merge.....fixed.